### PR TITLE
Autoconnect the next X speaker to jitsi

### PR DESCRIPTION
**Pull request by tsiegleauq**
In config, set "general_system_conference_auto_connect_next_speakers"
to let the next X speakers on the current list of speakers
automatically join the jitsi conference.
Updates automatically

### CODE REVIEW

File: client/src/app/site/agenda/models/view-list-of-speakers.ts [hunk 1]
```diff
@@ -65,8 +75,20 @@ export class ViewListOfSpeakers
         return this.finishedSpeakers.findIndex(speaker => speaker.user_id === checkSpeaker.user_id) !== -1;
     }
 
-    public isUserOnList(userId: number): boolean {
-        return !!this.speakers.find(speaker => speaker.user_id === userId);
+    public findUserIndexOnList(userId: number): number {
```
Comment (FinnStutzenstein) [inline client/src/app/site/agenda/models/view-list-of-speakers.ts:78]: You can use `UserListIndexType` as the return type.

**Review by FinnStutzenstein — APPROVED** [review]
Code looks good

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
Please don't autoconnect if user was started with "play" button on list of speaker.

Use case:
User A is on position 1 in the LOS. Finished his speach. Exit jitsi. Admin has forgot to click play and stop. Do it now. Now: User A autoconnects again to jitsi! - > Please prevent this!

Only the x users on the next speakers list should autoconnect to jitsi. Current speaker should do nothing.

**Review by emanuelschuetze — APPROVED** [review]

**Comment by emanuelschuetze** [comment]
fixed typo and config order. rebased.

**Pull request merged**